src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Internal/ModelBindingHelper.cs

```diff
@@ -120,41 +85,6 @@ public static class ModelBindingHelper
             params Expression<Func<TModel, object>>[] includeExpressions)
            where TModel : class
         {
-            if (model == null)
```

Comment:
In this particular case, the failure behaviour is slightly different but all `null` parameters are still caught. E.g. if `prefix` and `includeExpressions` were both `null`, would previously get an `ArgumentNullException` complaining about `prefix`. Would now be about `includeExpressions`. Could restore these checks if anyone thinks the order of these checks is important to preserve.